lib/multicell_location/services/nrf_cloud_integration.c

```diff
@@ -75,7 +75,10 @@ int location_service_get_cell_location(const struct lte_lc_cells_info *cell_data
 		.exp_delta_s = (5 * 60),
 		.sec_tag = CONFIG_MULTICELL_LOCATION_NRF_CLOUD_JWT_SEC_TAG,
 		.key = JWT_KEY_TYPE_CLIENT_PRIV,
-		.alg = JWT_ALG_TYPE_ES256
+		.alg = JWT_ALG_TYPE_ES256,
+		/* Set to NULL so a properly sized buffer is allocated */
+		.jwt_buf = NULL,
+		.jwt_sz = 0
```

Comment:
I believe unlisted members are initialized to zero, if a shorter initializer list is given.

but it does not harm to be verbal about it.